source/view.js

```diff
@@ -38,4 +40,4 @@ function View(){
             occupiedCells[i] = fieldCellsArr[i];
```

Comment:
Can we rename this function to make it describe its purpose better? updateCellsContents or just updateCells?